pkg/services/alerting/notifiers/dingding.go

```diff
@@ -61,15 +95,34 @@ func (this *DingDingNotifier) Notify(evalContext *alerting.EvalContext) error {
 		message = title
 	}
 
-	bodyJSON, err := simplejson.NewJson([]byte(`{
-		"msgtype": "link",
-		"link": {
-			"text": "` + message + `",
-			"title": "` + title + `",
-			"picUrl": "` + picUrl + `",
-			"messageUrl": "` + messageUrl + `"
-		}
-	}`))
+	for i, match := range evalContext.EvalMatches {
+		message += fmt.Sprintf("\\n%2d. %s: %s", i+1, match.Metric, match.Value)
+	}
+
+	var bodyStr string
+	if this.MsgType == "actionCard" {
+		bodyStr = `{
+			"msgtype": "actionCard",
+			"actionCard": {
```

Comment:
looks like the actionCard does not have a picUrl equivalent so changing to actionCard by default (or removing) link opton would be bad I guess